sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/query.sql

```diff
@@ -328,6 +328,7 @@ weather_events AS (
   SELECT
     mozfun.map.get_key(event_details, "newtab_visit_id") AS newtab_visit_id,
     COUNTIF(event_name = 'weather_impression') AS weather_widget_impressions,
+    COUNTIF(event_name = 'weather_location_selected') AS weather_widget_location_selected,
```

Comment:
@m-d-bowerman This needs to be added to the `ARRAY_AGG(STRUCT(...` in the `weather_summary` CTE to be in the output.  This is failing dry run right now

Reply:
@BenWu any ideas on why this passes local bqetl validation? isn't that a dry run?

Reply:
It does fail CI https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/37179/workflows/0fde07c7-2922-4e99-8261-a9a36f469f56/jobs/420000
But dry runs aren't required to pass before merge because they can be flaky so the auto-merge still went through